docs/UserGuide.md

```diff
@@ -20,20 +30,51 @@ Vax@NUS is a **one stop management app to efficiently track and schedule COVID-1
 
 1. Copy the file to the folder you want to use as the _home folder_ for your Vax@NUS application.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. 
```

Comment:
Should it be "similar to the one below" (i.e. add the word "one")?

Reply:
Added